rules/library.bzl

```diff
@@ -263,13 +264,42 @@ def _xcframework_slice(*, xcframework_name, identifier, platform, platform_varia
             identifier,
             xcframework_name,
         ))
-    return (platform, platform_variant, supported_archs, import_name)
+
+    # Collecting interface files for VFS
+    import_headers = native.glob(
+        ["%s/**/*.h" % path],
+    )
+    import_module_maps = native.glob(
+        ["%s/**/*.modulemap" % path],
+    )
+    import_swiftmodule_contents = native.glob(
+        ["%s/**/*.swiftmodule/*.*" % path],
+    )
+    if len(import_module_maps) > 0:
+        import_module_map = import_module_maps[0]
+    else:
+        import_module_map = None
+
+    vfs_imported_framework = _find_imported_framework_name(import_headers)
+    if vfs_imported_framework != None:
+        framework_vfs_overlay(
+            name = resolved_target_name_vfs_overlay,
+            framework_name = vfs_imported_framework,
+            modulemap = import_module_map,
+            swiftmodule = import_swiftmodule_contents,
```

Comment:
I'm wondering if we should add the `swiftmodule` attribute to all of the imports - it looks like we may be missing them